app/Exceptions/Handler.php

```diff
@@ -29,27 +32,24 @@ class Handler extends ExceptionHandler
     /**
      * Report or log an exception.
      *
-     * @param  \Throwable  $exception
-     * @return void
-     *
      * @throws \Exception
      */
-    public function report(Throwable $exception)
+    public function report(Throwable $exception): void
```

Comment:
Do we need this empty method here?

Reply:
We could use it for logger which is not implemented yet. Even if we implement logger, report method isn't necessary to log exceptions from handler. When we decide to use it, I'll just restore it.